pkg/strategy/bollmaker/strategy.go

```diff
@@ -653,6 +671,36 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 			return
 		}
 
+		// Update spreads
```

Comment:
please filter kline.Interval.  
In backtest, 1m is automatically subscribed (however, in runtime, only s.Interval is used)

Reply:
cc @c9s known differences between runtime and backtest.

Reply:
This the intended behavior. I use the same interval as the strategy

Reply:
I mean, in backtest, 1m and s.Interval are both subscribed. 1m is subscribed by the system automatically, and s.Interval is subscribed by the strategy.  

while in runtime, only s.Interval is subscribed.

Reply:
so better to move this after line 709

Reply:
I see what you mean. That might explain something